### PR TITLE
Web Device API reference update for Tizen 4.0

### PR DESCRIPTION
**Pull request by pwasowski2**
### Change Description ###

This commit updates and unifies style and formatting of Web Device API reference descriptions and code examples for Tizen 4.0.

### CODE REVIEW

File: docs/application/web/api/4.0/device_api/mobile/tizen/nfc.html [hunk 1]
```diff
@@ -854,17 +794,125 @@ <h4>Attributes</h4>
...
 <dl>
+<dt class="deprecated method" id="NFCAdapter::setPowered">
+<a class="backward-compatibility-anchor" name="::NFC::NFCAdapter::setPowered"></a><code><b><span class="methodName">setPowered</span></b></code>
+</dt>
+<dd class="deprecated">
+<div class="brief">
+ Sets the power of an NFC adapter to either an on state or an off state.
+            </div>
+<p class="deprecated"><b>Deprecated.</b>
+ Deprecated since 2.3. Instead, let the user turn NFC on/off through the Settings application. See the <a href="../../../../../org.tizen.tutorials/html/web/tizen/network/nfc_tutorial_w.htm#Managing_NFC_Conn">Managing NFC</a> Tutorial.
```
Comment (mijin85cho) [inline docs/application/web/api/4.0/device_api/mobile/tizen/nfc.html]: Please use the latest Tizen Developer Site link. This link is invalid now.

**Comment by pwasowski2** [comment]
Hyperlinks to _Managing Bluetooth Tutorial_ and _Managing NFC Tutorial_ have been updated.

File: docs/application/web/api/4.0/device_api/mobile/tizen/nfc.html (hunk 1 above)
Comment (pwasowski2) [reply docs/application/web/api/4.0/device_api/mobile/tizen/nfc.html]: It has been updated

File: docs/application/web/api/4.0/device_api/mobile/tizen/bluetooth.html [hunk 2]
```diff
@@ -1266,51 +1202,309 @@ <h4>Methods</h4>
...
+<dt class="deprecated method" id="BluetoothAdapter::setPowered">
+<a class="backward-compatibility-anchor" name="::Bluetooth::BluetoothAdapter::setPowered"></a><code><b><span class="methodName">setPowered</span></b></code>
+</dt>
+<dd class="deprecated">
+<div class="brief">
+ Sets the state of a Bluetooth adapter to on or off by sending a request to Bluetooth hardware to change the power state.
+For most Bluetooth actions, the Bluetooth adapter must be powered on.
+            </div>
+<p class="deprecated"><b>Deprecated.</b>
+ It is deprecated since Tizen 2.3. Instead, let the user turn on/off Bluetooth through the Settings application. See <a href="../../org.tizen.mobile.web.appprogramming/html/tutorials/communication_tutorial/managing_bluetooth.htm">Managing Bluetooth Tutorial</a>.
```
[the remark this answers is not in the document]
Comment (pwasowski2) [reply docs/application/web/api/4.0/device_api/mobile/tizen/bluetooth.html]: It has been updated

**Comment by mijin85cho** [comment]
Hello, @huytaduc.
Why is this build failed? Would you please give us the clue to solve it?

**Comment by mijin85cho** [comment]
Because of the schedule,
I'll merge this PR without @vincent-kwon's confirmation.
If you find issues in the future, please raise a new PR.

**Pull request merged**